integration/spark2/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala

```diff
@@ -42,7 +42,8 @@ trait ServerInterface {
   /**
    * Used to prune and cache the datamaps for the table.
    */
-  def getSplits(request: DistributableDataMapFormat): Array[ExtendedBlocklet]
+  def getSplits(request: DistributableDataMapFormat, taskGroupId: String, taskGroupDesc: String):
```

Comment:
Lets make taskGroupId and taskGroupDesc optional with "" as the default value.

Reply:
Done